### DISCUSSION TITLE
Does cdk8s typescript support argo workflows?

### DISCUSSION THREAD

**Discussion by zip-hashan** [cdk8s-team/cdk8s#1289] [Q&A]
Hi There,

I am wondering if there is typescript support for ARGO workflows in cdk8s? If, so where can I find the documentation?

Thanks in advance 🙏

**Comment by shinebayar-g** [comment]
By support you mean declaring the Argo CRDs? If yes, checkout https://cdk8s.io/docs/latest/cli/import/ . You can import CRDs and generate Typescript classes. However there are some unexpected behaviors like https://github.com/cdk8s-team/cdk8s/issues/883
